### PR TITLE
add new page course

### PR DESCRIPTION
**Pull request by Goven112**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by DimaDiD — APPROVED** [review]

File: src/models/Course/Course.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    name: string;
+    link: string;
+    
+  
+    constructor() {
+      this.id = 0;
+      this.name = "";
+      this.link = "";
+    }
+  }
```
Comment (Serhii-Buniak) [inline src/models/Course/Course.tsx:12]: In Typescript better use Interface if you don'n have any methods. And we try to always use interface in `/Models`

File: src/pages/userPage/personalData/Course.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  const getButton = async () => {
+    setvisibleAchievementModal(true);
+   
+  };
+
+  const fetchData = async () => {
+   // const response =  await getAllCourse();
+    const response1 = await getAllCourseByUserId(activeUserId);
+
+    console.log(response1);
```
Comment (Serhii-Buniak) [inline src/pages/userPage/personalData/Course.tsx:47]: Do not write `console.logs` and comments in pull request

**Review by Serhii-Buniak — APPROVED** [review]

File: src/models/Course/Course.tsx (hunk 1 above)
Comment (Serhii-Buniak) [inline src/models/Course/Course.tsx:12]: In Typescript better use Interface if you don'n have any methods. And we try to always use interface in `/Models`

File: src/pages/userPage/personalData/Course.tsx (hunk 2 above)
Comment (Serhii-Buniak) [inline src/pages/userPage/personalData/Course.tsx:47]: Do not write `console.logs` and comments in pull request

**Review by Serhii-Buniak — CHANGES_REQUESTED** [review]

File: src/models/Course/Course.tsx (hunk 1 above)
Comment (Goven112) [reply src/models/Course/Course.tsx:12]: but in folder models is class . i done the same

**Review by tvardero — APPROVED** [review]

**Pull request merged**